### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by fxcoudert** [wellington/wellington#213]
We're shipping the latest release of wellington in Homebrew, but now it doesn't build anymore since it's not compatible with latest go (while master is). Could you please cut a new release?

**Comment by drewwells** [comment]
I probably should bake go into the brew. Last I checked they don't use
repeatable builds in brew since I can't pick what version of Go is used.
Can you include what version of go it built with or include the error you
see?

**Comment by fxcoudert** [comment]
Homebrew is always using the latest go, so currently 1.11.1

```
# github.com/wellington/wellington/vendor/github.com/fsnotify/fsevents
vendor/github.com/fsnotify/fsevents/fsevents.go:38:31: constant 36893488147419103231 overflows uint64
vendor/github.com/fsnotify/fsevents/fsevents.go:236:44: cannot use nil as type _Ctype_CFAllocatorRef in argument to _Cfunc_CFStringCreateWithCString
vendor/github.com/fsnotify/fsevents/fsevents.go:237:163: cannot use cPaths (type _Ctype_CFArrayRef) as type _Ctype_CFMutableArrayRef in argument to func literal
vendor/github.com/fsnotify/fsevents/fsevents.go:240:38: constant 36893488147419103231 overflows _Ctype_ulonglong
```

**Comment by drewwells** [comment]
looks related to this commit https://github.com/wellington/wellington/commit/0b5ba4d9c70dedfb20915518c230f58008fd08d7

**Comment by drewwells** [comment]
did you install the published wellington or use the `--HEAD` command?

**Comment by drewwells** [comment]
I've shipped new code and changed my CI to test go 1.11. I didn't see any 1.11 specific issues, but there have been significant code changes. Can you please test it by trying this `brew install --HEAD wellington`?

If it looks good, I'll look into shipping a new release and adding it to homebrew

**Comment by Bo98** [comment]
Hi @drewwells, do you still plan a new release? Wellington is one of just **9** formulae that have not been able to be built since Mojave's release (Sept 2018).
